addons/test_mail/tests/test_message_track.py

```diff
@@ -314,6 +320,21 @@ def test_tracked_compute(self):
             ('partner_email', 'char', 'foo@example.com', 'bar@example.com'),
         ])
 
+    def test_message_track_author(self):
```

Comment:
As tests are executed alphabetically, could you order test method alphabetically in a given class ? That way the output of tests follows the test ordering, which is convenient to use :)